src/native/corehost/hostpolicy/hostpolicy_context.cpp

```diff
@@ -119,6 +119,23 @@ namespace
         {
             return pal::pal_utf8string(get_filename_without_ext(context->application), value_buffer, value_buffer_size);
         }
+        else if (::strcmp(key, HOST_PROPERTY_COMMAND_LINE) == 0)
+        {
+            pal::string_t command_line = context->host_path;
+            if (context->host_mode == host_mode_t::muxer)
+            {
+                command_line.append(_X(" "));
+                command_line.append(context->application);
+            }
+
+            for (int i = 0; i < context->app_argc; ++i)
+            {
+                command_line.append(_X(" "));
```

Comment:
This does not do any escaping. It means that this can be used as an informational string for diagnostic purposes, but it cannot be used for anything that needs to be precise or that needs to roundtrip. If this stays as is, we may want to reflect it in the name `COMMAND_LINE_FOR_DIAGNOSTIC_PURPOSES_ONLY` to make it clear that it is not accurate representation of the command line.